src/test/resources/emissary/test/core/schemas/answers.xsd

```diff
@@ -0,0 +1,604 @@
+                <xs:element ref="name"/>
+                <xs:element ref="value"/>
+            </xs:sequence>
+            <xs:attribute ref="os-release"/>
+            <xs:attribute ref="os-version"/>
+            <xs:attribute ref="matchMode"/>
+            <xs:attribute ref="collectionSeparator"/>
+        </xs:complexType>
+    </xs:element>
+    <xs:element name="noview">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element ref="name"/>
+            </xs:sequence>
+            <xs:attribute ref="os-release"/>
+            <xs:attribute ref="os-version"/>
+        </xs:complexType>
+    </xs:element>
+
+    <xs:attribute name="os-release">
+        <xs:simpleType>
+            <xs:restriction base="xs:string">
+                <xs:enumeration value="rhel"/>
+                <xs:enumeration value="centos"/>
+                <xs:enumeration value="ubuntu"/>
+                <xs:enumeration value="mac"/>
+            </xs:restriction>
+        </xs:simpleType>
+    </xs:attribute>
+    <xs:attribute name="os-version" type="xs:string" />
+    <xs:attribute name="matchMode">
+        <xs:simpleType>
+            <xs:restriction base="xs:string">
+                <xs:enumeration value="equals"/>
+                <xs:enumeration value="index"/>
+                <xs:enumeration value="!index"/>
+                <xs:enumeration value="contains"/>
+                <xs:enumeration value="!contains"/>
+                <xs:enumeration value="match"/>
+                <xs:enumeration value="base64"/>
+                <xs:enumeration value="collection"/>
+            </xs:restriction>
+        </xs:simpleType>
+    </xs:attribute>
+    <xs:attribute name="collectionSeparator" type="xs:string" />
+
+    <xs:complexType name="baseType" mixed="true">
+        <xs:attribute ref="os-release"/>
+        <xs:attribute ref="os-version"/>
+    </xs:complexType>
+    <xs:complexType name="child">
+        <xs:choice minOccurs="0" maxOccurs="unbounded">
+            <xs:element ref="currentForm"/>
+            <xs:element ref="currentFormSize"/>
+            <xs:element ref="fileType"/>
+            <xs:element ref="classification"/>
+            <xs:element ref="shortName"/>
+            <xs:element ref="fontEncoding"/>
+            <xs:element ref="broken"/>
+            <xs:element ref="procError"/>
+            <xs:element ref="meta"/>
+            <xs:element ref="nometa"/>
+            <xs:element ref="data"/>
+            <xs:element ref="dataLength"/>
+            <xs:element ref="view"/>
+            <xs:element ref="noview"/>
+            <xs:element ref="numAttachments"/>
+            <xs:element ref="extractCount"/>
+            <xs:group ref="children"/>
+        </xs:choice>
+        <xs:attribute ref="os-release"/>
+        <xs:attribute ref="os-version"/>
+    </xs:complexType>
+    <xs:group name="children">
+        <xs:sequence>
+            <xs:element name="att1" type="child" minOccurs="0"/>
```

Comment:
Could we add another element here named `att` that's a different type that could be validated independently much like the current child; that way we can have a `maxOccurs="unbounded"` and eventually move to that strategy over time?

Reply:
Added the ability to specify the elements as:
```
<att index="1">
<extract index="1">
```
Updated ExtractionTest class to support.